src/game/camera.rs

```diff
@@ -161,9 +180,10 @@ mod test {
         assert_eq!(camera.roll(), 3.0);
         assert_eq!(camera.position(), &Vec3f::new(4.0, 5.0, 6.0));
 
-        assert!(camera.modelview().approx_eq(
-                &(Mat4::new_euler_rotation(1.0, 2.0, 3.0)
-                  * Mat4::new_translation(Vec3f::new(-4.0, -5.0, -6.0))),
-                1e-16));
+        assert!(camera.modelview().approx_eq(&(Mat4::new_euler_rotation(1.0, 2.0, 3.0) *
+                                               Mat4::new_translation(Vec3f::new(-4.0,
+                                                                                -5.0,
+                                                                                -6.0))),
+                                             1e-16));
```

Comment:
Don't mind the drift as much as the splitting the vec construction `Vec3f` on three lines. That said, it's a test, I won't lose any sleep over it :)